sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -90,27 +92,13 @@ dynamic widget = response.Content.ToDynamicFromJson();
 string id = widget["$id"];
 ```
 
-### Working with Azure values
-
-When working with JSON from Azure services, you can learn what properties are available in the JSON response content from the REST API documentation for the service, examples in the protocol method documentation, or by expanding the [Dynamic View](https://learn.microsoft.com/visualstudio/debugger/watch-and-quickwatch-windows) in Visual Studio.
-
-```C# Snippet:AzureCoreUseDynamicDataDefaults
-Response response = client.GetWidget();
-dynamic widget = response.Content.ToDynamicFromJson(DynamicDataOptions.Default);
-string id = widget.Id;
-widget.Name = "New Name";
-client.SetWidget(RequestContent.Create(widget));
-```
-
-If no parameter is passed to `ToDynamicFromJson()`, properties names must exactly match the member names in the JSON content.  Passing `DynamicDataOptions.Default` will enable properties to be accessed using PascalCase property names, and will write any added properties with camelCase names.
```

Comment:
Should we have a note at the end of this document explaining how the default mapping works?

Reply:
Added.